seqr/utils/elasticsearch/es_utils_2_3_tests.py

```diff
@@ -1194,9 +954,10 @@ def assertSameSearch(self, executed_search, expected_search_params):
         if expected_search_params['filters']:
             expected_search['query'] = {
                 'bool': {
-                    'filter': expected_search_params['filters']
+                    'filter': sort_filter(expected_search_params['filters'])
```

Comment:
`expected_search_params` are created as synthetic data in the test. Just put them in in the correct order, don't post process them

Reply:
Order changes to the expected data will be hard to review. Are you sure that you want that?

Reply:
I have removed both sorting for executed and expected filters by adding a few more `sorted()` in the filter creation codes.